test/merchant_test.rb

```diff
@@ -1,34 +1,27 @@
 require 'simplecov'
 SimpleCov.start
-require './lib/merchant'
-require 'rspec'
-
-RSpec.describe Merchant do
+require './lib/helper'
+
+RSpec.describe InvoiceRepository do
+  let!(:sales_engine) {SalesEngine.from_csv({
+    :items     => "./data/items.csv",
+    :merchants => "./data/merchants.csv"
+  })}
+  let!(:merchant_repository) {sales_engine.merchants}
+  let!(:merchant) {merchant_repository.create(Merchant.new({
+    :id => 5,
+    :name => "Turing School"
+  }))}
 
   it 'exists' do
-    merchant = Merchant.new({
-      :id => 5,
-      :name => "Turing School"
-      })
-
-    expect(merchant).to be_instance_of(Merchant)
+    expect(merchant_repository).to be_instance_of(Merchant)
   end
 
   it 'returns id' do
```

Comment:
Getting 'undefined method 'id' for #<Array>:blahblahblah'

![image](https://user-images.githubusercontent.com/17027357/171972009-683e449e-03d1-4a10-9dce-166ea04d3023.png)

Reply:
fixed